demos/demo_references.bib

```diff
@@ -25,6 +25,14 @@ @InBook{2015:lmscup
                   {http://www1.maths.leeds.ac.uk/~matak/lms-cup2015.pdf}
 }
 
+@Manual{Gidel:2017,
```

Comment:
Ok then I apologise. The ref. Gidel:2017 is an article which is accepted by Nonlinear Processes Geophysics.